### PR TITLE
Update the sequence of commands in install-on-linux.md

### LINKED ISSUE #157: Fix Install sequence for Redis Enterprise Software on Linux
dpkg-sig --verify  need to run on installation package (*.deb) which is only available after
```
sh tar vxf <tarfile name>
```

Can we fix this small change?

PR: https://github.com/redis/docs/pull/155

### PR DESCRIPTION
**Pull request by grvsoni**
dpkg-sig --verify  need to run on installation package (*.deb) which is only available after
```
sh tar vxf <tarfile name>
```

Fixes #157.

### CODE REVIEW

**Comment by dwdougherty** [comment]
@rrelledge This LGTM, but since it's your area, can you review and approve?

File: content/operate/rs/installing-upgrading/install/install-on-linux.md [hunk 1]
```diff
@@ -16,11 +16,17 @@ For installation on machines without an internet connection, see [Offline instal
 
 ## Install on Linux
 
-To install Redis Enterprise Software using the command line:
+To install Redis Enterprise Software use the command line:
+
```
Comment (rrelledge) [inline content/operate/rs/installing-upgrading/install/install-on-linux.md:20]:
```suggestion

1. Copy the installation package to the node.

```

File: content/operate/rs/installing-upgrading/install/install-on-linux.md [hunk 2]
```diff
@@ -16,11 +16,17 @@ For installation on machines without an internet connection, see [Offline instal
...
+1. On the node, change to the directory where the installation package is located and extract the installation files:
+
+    ```sh
+    tar vxf <tarfile name>
+    ```
 
 1. Copy the installation package to the node.
```
Comment (rrelledge) [inline content/operate/rs/installing-upgrading/install/install-on-linux.md:27]:
```suggestion
```

File: content/operate/rs/installing-upgrading/install/install-on-linux.md [hunk 3]
```diff
@@ -16,11 +16,17 @@ For installation on machines without an internet connection, see [Offline instal
...
 
-    _(Optional)_ Use the {{< download "GPG key file" "../GPG-KEY-redislabs-packages.gpg" >}} to confirm authenticity of Ubuntu/Debian or RHEL RPM packages:
+    _(Optional)_ Use the {{< download "GPG key file" "../GPG-KEY-redislabs-packages.gpg" >}} to confirm the authenticity of Ubuntu/Debian or RHEL RPM packages:
```
Comment (rrelledge) [inline content/operate/rs/installing-upgrading/install/install-on-linux.md:29]:
```suggestion
1.  _(Optional)_ Use the {{< download "GPG key file" "../GPG-KEY-redislabs-packages.gpg" >}} to confirm the authenticity of Ubuntu/Debian or RHEL RPM packages:
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Thanks, @grvsoni. I suggest keeping "Copy the installation package to the node" as the first step before extracting the installation files, and then making the optional GPG section its own numbered step.

File: content/operate/rs/installing-upgrading/install/install-on-linux.md [hunk 4]
```diff
@@ -16,11 +16,17 @@ For installation on machines without an internet connection, see [Offline instal
 
 ## Install on Linux
 
-To install Redis Enterprise Software using the command line:
+To install Redis Enterprise Software use the command line:
```
Comment (rrelledge) [inline content/operate/rs/installing-upgrading/install/install-on-linux.md:19]:
```suggestion
To install Redis Enterprise Software, use the command line:
```

**Review by dwdougherty — APPROVED** [review]
LGTM. Thank you, @rrelledge, for the review.

**Pull request merged**